polyfill/test/zoneddatetime.mjs

```diff
@@ -730,6 +730,23 @@ describe('ZonedDateTime', () => {
     });
   });
 
+  describe('Maths', () => {
+    const earlier = ZonedDateTime.from('1976-11-18T15:23:30.123456789-03:00[America/Santiago]');
+    const later = ZonedDateTime.from('2019-10-29T10:46:38.271986102-03:00[America/Santiago]');
+    ['years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'].forEach((largestUnit) => {
+      const diff = later.since(earlier, { largestUnit });
+      it(`earlier.since(later, ${largestUnit}) == later.since(earlier, ${largestUnit}).negated()`, () =>
+        equal(`${earlier.since(later, { largestUnit })}`, `${diff.negated()}`));
+      it(`earlier.until(later, ${largestUnit}) == later.since(earlier, ${largestUnit})`, () =>
```

Comment:
Same here. Because we chose (A), this test won't work for all values of earlier and later if months or years are involved. This is the exact test that (B) would make pass in all cases.